scss/_card.scss

```diff
@@ -364,7 +364,7 @@
   }
 
   .card-body {
-    padding: 0 1.5rem 0 0;
+    padding: 0 1.5rem 0 .125rem;
```

Comment:
I doesn't seem to be well left-aligned with `button, a` and `.btn-sm`. But it is better for `.btn-lg`.

Reply:
🤔 can't remember why I've added that padding. But you're right this break the alignement. Maybe need to revert this line

Reply:
Yep, I would revert this line.

Reply:
reverted